builder/chroot-script.sh

```diff
@@ -1,51 +1,172 @@
 #!/bin/bash
 set -ex
 
+# save HYPRIOT_* content from /etc/os-release. Needs to be done in
+# case base-files gets updated during build
+< /etc/os-release grep HYPRIOT_ > /tmp/os-release.add
```

Comment:
This is requires because we are currently picking up os-rootfs 1.0 - and there have been changes to base-files since its release, which overwrites /etc/os-release. Second part is on #L163